src/sssom/cli.py

```diff
@@ -253,14 +254,23 @@ def split(input: str, output_directory: str):
 @input_argument
 @output_option
 @click.option("-W", "--inverse-factor", help="Inverse factor.")
-def ptable(input, output: TextIO, inverse_factor):
+@click.option(
+    "--default-confidence",
+    default=None,
+    help="Default confidence to be assigned if absent.",
+)
+def ptable(input, output: TextIO, inverse_factor: float, default_confidence: float):
     """Convert an SSSOM file to a ptable for kboom/`boomer <https://github.com/INCATools/boomer>`_."""
     # TODO should maybe move to boomer (but for now it can live here, so cjm can tweak
     msdf = parse_sssom_table(input)
     # df = parse(input)
+    if default_confidence:
+        msdf.df = add_default_confidence(msdf.df, default_confidence)
```

Comment:
Why is this added twice? Once here, and once in `dataframe_to_ptable`?

Reply:
Yeah I had that question too... this is how it was from the beginning. Let me streamline it.

Reply:
Done

Reply:
Great.